Cookbook/Technical-Documents/BuildTime-Spikes-Report.md

```diff
@@ -0,0 +1,190 @@
+# Improving our Build Times
+
+Build Times in Swift are known to be quite slow, but we could still try to find ways to reduce them. So far we've tried the following different tactics to reduce our build times especially on CI and clean builds:
+
+ - [➥](#helping-the-type-checker)
```

Comment:
I mean all these tickets can be part of epic and in the epic or individual tickets we have report with results

Reply:
This report and its exposition to the Playbook has been originally requested by @dmcrodrigues , and I agree with him too that we should have that exposed as a document, and not as individual reports in the Epic.

- The goal is to have clear reports of all the things we try in an *accessible* document in case we want to try one of these solutions later and want to remember what we tried, what went wrong, what might have evolved since last time. And I'm pretty sure we'll never think about searching JIRA to try to remember which solution we tried (and all solutions we tried) and what to search. So a single document is better for the future in that regard
 - Then it could indeed be a Jira/Wiki article instead, but I'm not sure I see the point of not exposing that in our playbook. After all, it's open information and valuable experience that we have no reason to hide in private wikis anyway.

Reply:
I totally agree with this. We should definitely expose our findings. It will be beneficial to the whole community 🙂